apps/desktop-wallet/electron/utils.ts

```diff
@@ -9,4 +11,15 @@ export const CURRENT_VERSION = app.getVersion()
 export const IS_RC = CURRENT_VERSION.includes('-rc.')
 
 // See: https://www.electronjs.org/docs/latest/tutorial/security#17-validate-the-sender-of-all-ipc-messages
-export const isIpcSenderValid = (frame: WebFrameMain | null) => frame && new URL(frame.url).host === 'localhost:3000'
+export const isIpcSenderValid = (frame: WebFrameMain | null) => {
+  if (frame) {
+    const url = new URL(frame.url)
+
+    return (
+      url.protocol === `${APP_PROTOCOL}:` || // for production builds
+      url.host === 'localhost:3000' // for dev environment
```

Comment:
This is the reason why all IPC calls were filtered out. The production build was using the `file://` protocol, which was filtered out. Instead of allowing the `file://` protocol to pass, I followed the best security practices and replaced it with a custom app protocol `alephium://`.